src/mjolnir/graphfilter.cc

```diff
@@ -31,6 +31,26 @@ uint32_t can_aggregate = 0;
 // Group wheelchair and pedestrian access together
 constexpr uint32_t kAllPedestrianAccess = (kPedestrianAccess | kWheelchairAccess);
 
+/**
+ * Is there an opposing edge with matching edgeinfo offset. The end node of the directed edge
+ * must be in the same tile as the directed edge.
```

Comment:
but shouldnt we still throw if someone calls it with an edge that leaves the tile?